common/hooks/post-install/11-pkglint-elf-in-usrshare.sh

```diff
@@ -18,7 +18,11 @@ hook() {
         case "${mime}" in
             # Note application/x-executable is missing which is present in most Electron apps
             application/x-sharedlib*|application/x-pie-executable*)
-                matches+=" ${file#$PKGDESTDIR}" ;;
+                if [[ ${ignore_in_usr_share} != *$(basename "${file}")* ]]
```

Comment:
Can't you just compare it to `${file##/*}`, is faster but should give the same output as basename. (find does afaik never give you a path with a trailing `/`)